buildah.go

```diff
@@ -97,6 +97,8 @@ type Builder struct {
 	Docker docker.V2Image `json:"docker,omitempty"`
 	// DefaultMountsFilePath is the file path holding the mounts to be mounted in "host-path:container-path" format
 	DefaultMountsFilePath string `json:"defaultMountsFilePath,omitempty"`
+	// Adds a trigger instruction to be executed when the image is used as the base of another image.
+	OnBuild           []string            `json:"OnBuild,omitempty"`
```

Comment:
No need for a specific field here - the value should be accessible in `Docker.Config.OnBuild`.

Reply:
Removed this too.   Hmm, what happened to my comments GitHub? ...